grails-app/views/layouts/_tracking.gsp

```diff
@@ -0,0 +1,12 @@
+<g:if env="production">
```

Comment:
This `if` clause needs to be tested

Reply:
@Aapzu did you test it?

Reply:
Decided with @hpihkala that it's not worth it to test it. Let's fix it if it doesn't work in production